### DISCUSSION TITLE
关于米游铺

### DISCUSSION THREAD

**Discussion by hot-ice-seele** [share121/inter-knot#5239] [其他]
米游铺上买的周边一般多久可以发货？
求好心人解答(◍•ᴗ•◍)❤

**Comment by HLXS-HLXS** [comment]
快的第二天就发了，慢的能给你拖一个星期。如果你下单的时候是多样物品一起下单，且其中有预售商品的话，那他会等这个预售商品到货了再发货，而不是现货预售分开发

**Reply by hot-ice-seele** [reply]
清楚了，谢谢
